packages/mantine/src/components/badge/Badge.tsx

```diff
@@ -102,6 +102,21 @@ const BadgeSecondary = enhanceBadge(
         bg: alpha('var(--mantine-color-gray-3)', 0.16),
     }),
 );
+const BadgeSuccess = enhanceBadge(
+    MantineBadge.withProps({
+        variant: 'light',
+        color: 'lime',
+        bd: '1px solid var(--badge-bg)',
+        c: 'lime.6',
+    }),
+    MantineBadge.withProps({
+        variant: 'light',
+        color: 'lime',
+        c: 'lime.2',
+        bd: `1px solid ${alpha('var(--mantine-color-lime-3)', 0.16)}`,
+        bg: alpha('var(--mantine-color-lime-3)', 0.16),
```

Comment:
There seem to be an issue with the lime palette, in dark mode it's not green at all

<img width="278" height="50" alt="Image" src="https://github.com/user-attachments/assets/8efba6f6-e8cd-41c6-9ca6-0f7a555beab9" />

Edit. it seems to be this: https://github.com/coveo/plasma/blob/f81589b1c49ec25ef67d14e5c294f7c9e11e6be5/packages/mantine/src/theme/PlasmaColors.ts#L11-L22

We should make sure the lime palette as 10 values

Reply:
woops, yep